functions/Remove-SqlOrphanUser.ps1

```diff
@@ -257,6 +363,7 @@ Will remove from all databases the user OrphanUser EVEN if exists their matching
 		if ($StackSource -ne "Repair-SqlOrphanUser")
         {
            Write-Output "Total Elapsed time: $totaltime"
+           return $results
```

Comment:
if all objects are streamed into the pipeline directly, no return $results is needed